### DISCUSSION TITLE
Export to excel doesn't work

### DISCUSSION THREAD

**Discussion by Mirka11** [DaxStudio/DaxStudio#1019] [General]
I'm just starting with DAX Studio and I was testing different export options to export results. Unfortunetly I got stuck on excel export because it doesn't work. I can't find the reason in log, it is showing only following message. I'm unable to conclude what the real issue is. Did I missed something in configuration?
![image](https://user-images.githubusercontent.com/49652583/211190987-7c6816a0-d2eb-4554-8b23-e107fab4ae15.png)

**Comment by dgosbell** [comment]
I have not seen anything like this before, it looks like something is blocking Excel from telling DAX Studio which workbook is currently active. Do you have any dialog boxes open like the file save/open dialogs or the Power Pivot management window?

Could you try restarting Excel with the diagnostic logging option (see https://daxstudio.org/docs/troubleshooting/logging-settings/#enabling-diagnostic-logging---excel-addin )  and then re-create this error and the close DAX Studio and Excel and attach the log file to this issue in Github?
